install/kubernetes/helm/istio/charts/grafana/values.yaml

```diff
@@ -4,7 +4,7 @@
 enabled: false
 replicaCount: 1
 image:
-  repository: grafana/grafana
+  repository: "{{ .Values.hub }}/grafana/grafana"
```

Comment:
change to:
```
hub: docker.io/grafana
```

then change grafana/templates/deployment.yaml to:
```
image: "{{ .Values.hub }}/grafana:{{ .Values.tag }}"
```

Should give you what you want